### ISSUE TITLE
EventsState should handle task-sent event from the client (not worker) correctly

### ISSUE THREAD

**Issue by jackyfkc** [mher/flower#435]
If the directive CELERY_SEND_TASK_SENT_EVENT was enabled, the client (like the beat) will also send events (task-sent), and the event details are different from the workers'

In this case, EventsState.event() should check if the events received are from workers. Otherwise the latter render of dashboard page would failed due to the mismatched event details.

File events.py, Line 42, 
```python
self.counter[worker_name][event_type] += 1
```

should be changed to 
```python
if event_type != 'task-sent':
    self.counter[worker_name][event_type] += 1
```

**Comment by mher** [comment]
Can you describe in more details what is not working? How to reproduce? I've tried sending tasks with `beat` and didn't notice anything wrong. task-sent event shouldn't have any effect on dashboard. It is not used in event calculations.

**Comment by jackyfkc** [comment]
Sorry, I made a mistake, enable CELERY_SEND_TASK_SENT_EVENT will only cause the clients shown in the worker table on dashboard page(related to issue #387)

And I found you have fixed it in #417. But it might not be a proper fix, since the root reason is that we added the clients (not workers) to EventsState.counter.

**Comment by mher** [comment]
Please try the fix
